scripts/apps/master-desk/components/AssignmentsComponent.tsx

```diff
@@ -69,7 +69,10 @@ export class AssignmentsComponent extends React.Component<IProps, IState> {
         let total = 0;
 
         if (this.state.deskData && this.state.deskData[desk._id]) {
-            this.state.deskData[desk._id].forEach((item) => total += item ? item.count : 0);
+            this.state.deskData[desk._id].forEach((item) =>
+                this.assignmentStages.some((stage) =>
```

Comment:
Why are you using `some` when its return value is never used? `some` stops iterating if `true` is returned. If you only care about one item, it would be more clear to use `find`.